### PR TITLE
itR-7: feat: add router

### PR DESCRIPTION
**Pull request by glebpigulevsky**

### CODE REVIEW

File: src/components/MenuItem/MenuItem.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.navigation-link {
+  text-decoration: none;
+  color: aliceblue;
+}
+a:active {
```
Comment (glebpigulevsky) [inline src/components/MenuItem/MenuItem.css:6]: empty line

**Comment by glebpigulevsky** [comment]
what is about tests?

File: src/components/MenuItem/__spec__/menuItem.spec.tsx [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-  render(<MenuItem item="string" />);
-  const menuItemElement = screen.getByTestId("menu-item");
-
-  expect(menuItemElement).toBeInTheDocument();
+  <BrowserRouter>
+    render(
+    <MenuItem title={""} path={""} />
+    ); const menuItemElement = screen.getByTestId("menu-item");
+    expect(menuItemElement).toBeInTheDocument();
+  </BrowserRouter>;
```
Comment (glebpigulevsky) [inline src/components/MenuItem/__spec__/menuItem.spec.tsx:16]: that is wrong code. you have to put BrowserRouter inside of the render method. and js code outside of BrowserRouter Wrapper. this test doesnt work! same for others tests

File: src/components/MenuItem/MenuItem.css (hunk 1 above)
Comment (MilaKolasava) [reply src/components/MenuItem/MenuItem.css:6]: done

File: src/components/MenuItem/__spec__/menuItem.spec.tsx (hunk 2 above)
Comment (MilaKolasava) [reply src/components/MenuItem/__spec__/menuItem.spec.tsx:16]: But this test is working!

**Pull request closed without merging**